src/Controller/SecurityController.php

```diff
@@ -59,16 +59,4 @@ public function login(
             'error' => $helper->getLastAuthenticationError(),
         ]);
     }
-
-    /**
-     * This is the route the user can use to logout.
-     *
-     * But, this will never be executed. Symfony will intercept this first
-     * and handle the logout automatically. See logout in config/packages/security.yaml
-     */
-    #[Route('/logout', name: 'security_logout')]
-    public function logout(): void
-    {
-        throw new \Exception('This should never be reached!');
-    }
```

Comment:
Still might be good for newcomers? It explains how the feature works internally

Reply:
The problem is that this "fake" logout route always felt weird for newcomers and created friction. I think it's better to just forget about this. If someone needs to do something advanced related to logout, they'll just find plenty of information about this in the Symfony Docs. Thanks!

Reply:
Yeah, probably makes sense 👍